Exiled.Hints/CustomHints/HintPatch.cs

```diff
@@ -0,0 +1,40 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="HintPatch.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+using Hints;
+
+namespace Exiled.API.Features.CustomHints
+{
+    using System;
+    using System.Collections.Generic;
+
+    using HarmonyLib;
+    using Hints;
+    using Mirror;
+
+    /// <summary>
+    /// The patch for hints.
+    /// </summary>
+    [HarmonyPatch(typeof(HintDisplay))]
+    [HarmonyPatch("Show")]
+    public static class HintPatch
```

Comment:
both attributes can be in the same one `[HarmonyPatch(typeof(HintDisplay), nameof(HintDisplay.Show))]`